bower.json

```diff
@@ -2,10 +2,8 @@
   "name": "angular-google-places-autocomplete",
   "version": "0.2.4",
   "main": [
-      "/src/autocomplete.js",
-      "/src/autocomplete.css",
-      "/dist/autocomplete.min.js",
-      "/dist/autocomplete.min.css"
```

Comment:
Actually you can leave the dist folder and remove the src, to consume the minified versions.